.github/workflows/check.yml

```diff
@@ -67,6 +72,90 @@ jobs:
             rust-version:  ${{ needs.get-rust-versions.outputs.stable }}
             cache: true
 
+    check-pretty-printers:
+        needs: [ calculate-git-info, get-rust-versions ]
+        strategy:
+            # `fromJSON` is used here to convert string output to boolean
+            fail-fast: ${{ fromJSON(needs.calculate-git-info.outputs.is_bors_branch) }}
+            matrix:
+                # TODO: check on macos as well
+                os: [ ubuntu-latest, windows-latest ]
+                # `fromJSON` is used here to convert string output to sequence.
+                # `all` is a string with list of old, stable and nightly versions.
+                # Make sequence from two outputs with version is not possible here.
+                # TODO: check on more rustc versions
```

Comment:
It's still relevant - I would be nice to check pretty printers with Rust 1.66 since there were some major changes in 1.67 and it's quite easy to break something for older versions